.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "test/audio"]
 	path = test/audio
 	url = https://github.com/MTG/essentia-audio
-[submodule "essentia-models"]
+[submodule "essentia/models"]
```

Comment:
If we want to store the models inside `test` folder, the name of submodule should better be `test/models`. Also missing a  `git add` for this `test/models` folder.